Controller/TaskController.php

```diff
@@ -112,7 +110,7 @@ public function __construct(
         RestHelperInterface $doctrineRestHelper,
         TaskManagerInterface $taskManager,
         EntityManagerInterface $entityManager,
-        SerializerInterface $serializer,
```

Comment:
Is the serializer even used anymore? Shouldn't we then also add a deprecation message when the service is not null

Reply:
Added the deprecation message